### ISSUE TITLE
Add tooltip to other inventory type (ie taxlots when on the properties list) columns stating they cannot be sorted in this view.

### ISSUE THREAD

**Issue by haneslinger** [SEED-platform/seed#3670]

**Comment by RDmitchell** [comment]
@haneslinger -- hopefully we can fix this in the future? Is there already an issue to enable sorting for the "other" inventory type?

**Comment by haneslinger** [comment]
yes, but it's a heavy lift and I don't believe it's currently prioritized. @isalanglois @nllong, where is adding in other inventory type sorting on the road map?
